lib/beaker/dsl/install_utils.rb

```diff
@@ -826,7 +826,7 @@ def install_puppet_from_msi( host, opts )
           on host, "if not exist #{host['distmoduledir']} (md #{host['distmoduledir']})"
         end
 
-        on host, "msiexec /qn /i #{dest}"
+        on host, "cmd /C 'start /w msiexec.exe /qn /i #{dest}'"
```

Comment:
`#{dest}` or `#{link}`? The `curl` command above drops the `#{dest}` portion, but this call is specifically using it.